sound/soc/sof/ipc3-topology.c

```diff
@@ -519,6 +519,7 @@ static int sof_ipc3_widget_setup_comp_mixer(struct snd_sof_widget *swidget)
 static int sof_ipc3_widget_setup_comp_pipeline(struct snd_sof_widget *swidget)
```

Comment:
style: given this is SOF common code, so could say "for example: Intel TGL has, .."